tests/system/all_sets.cfg

```diff
@@ -85,6 +85,25 @@ backend = "mpl"
 results_dir = "all_sets_results_test"
 debug = True
 
+[lat_lon_vector]
+sets = ["lat_lon_vector"]
+case_id = "ERA-Interim"
+variables = ["U", "V"]
+seasons = ["ANN"]
+plevs = [850.0]
+contour_levels = [240, 245, 250, 255, 260, 265, 270, 275, 280, 285, 290, 295]
+diff_levels = [-10, -7.5, -5, -4, -3, -2, -1, -0.5, 0.5, 1, 2, 3, 4, 5, 7.5, 10]
+backend = "mpl"
+results_dir = "all_sets_results_test"
+debug = True
+# TODO: Replace with paths to external testing directory accessible by CircleCI.
```

Comment:
What directory should I use for the `data_path` variables? It would need to be something easily accessible or downloadable (e.g., wouldn't ask for a password on every use).

Reply:
Can we follow what CDAT does? https://github.com/CDAT/cdat_info/blob/4d35272dab39d162ae1cd9d99a6aa03e5dbdb1b9/cdat_info/cdat_info_src.py#L108

We can place our testing data on our data server at anl. 
https://web.lcrc.anl.gov/public/e3sm/
More info: https://acme-climate.atlassian.net/wiki/spaces/ED/pages/707002387/E3SM+Input+Data+Servers

@zshaheen I vaguely remember you started similar effort making CI work. Could you chime in on this? Thanks

Reply:
Hmm, I don't recall anything like what you're asking about.

Like what you said, I remember trying to setup the test data on another DOE machine and have all of the downloading be from there. I tired to get something similar to how MPAS-Analysis did it. The data is broken into chunks and transferred over piecewise, allowing for a cool download bar denoting the progress.

Reply:
See lines 60-61 in `tests/system/test_diags.py`. I will update the data path to be Anvil once I get access to that machine. Alternatively, we could host the test data on Cori. 

Also, it might be a good idea to have another pull request to move all the other `nc` files out of this repo and onto Anvil/Cori, since that would be outside the scope of this PR. I know #303 requires removing the `nc` files from the entire git history (i.e., not just the current state), but I believe it would be good to make sure the repo has no `nc` files in its current state before attempting that.

Reply:
@chengzhuzhang Yeah the paths are set in the `cfg` files, so I just added a test data path and set the test to copy over the data to that path. Right now, it's set to copy from another of my directories outside the repo. So, the "copy over" step (lines 60-61 in `tests/system/test_diags.py`) needs to be changed to copy down from Anvil instead.

Reply:
Yeah guys sorry, I don't really recall much regarding this. It's been some time.